assets/scss/templates/_notices.scss

```diff
@@ -0,0 +1,108 @@
+/* notices */
+
+@mixin notice-border($color) {
+  // kept in case we change our mind later
+  //border-left: 5px solid $color;
+}
+
+div.notices {
```

Comment:
Same, do we really need this div selector?

Reply:
I guess not
```suggestion
.notices {
```